app/models/user.rb

```diff
@@ -1261,7 +1261,7 @@ def create_user_profile
   end
 
   def set_random_avatar
-    if SiteSetting.selectable_avatars_enabled?
+    unless SiteSetting.selectable_avatars_enabled == "none"
```

Comment:
Writing the condition using the "positive" style feels easier to read

```suggestion
    if SiteSetting.selectable_avatars_enabled != "none"
```